bin/ci-expected-results

```diff
@@ -70,6 +70,16 @@ if [ "$actual_author_non_authorline_title" != "$expected_author_non_authorline_t
     exit 2
 fi
 
+actual_total=$(curl -v "localhost/search?for=rheumatoid%20arthritis%20(RA)&page=1&per-page=10&sort=relevance&order=desc&use-date=default" | jq ".total")
```

Comment:
As `actual_total` is used earlier in the script, I'd consider renaming this to something more representative.